src/Gelf/Logger.php

```diff
@@ -60,7 +60,11 @@ public function __construct(
      */
     public function log($level, $rawMessage, array $context = array())
     {
-        $message = $this->initMessage($level, $rawMessage, $context);
+        if (!($rawMessage instanceof MessageInterface)) {
+            $message = $this->initMessage($level, $rawMessage, $context);
+        } else {
+            $message = $rawMessage;
+        }
```

Comment:
The longer I think about it - this does not make any sense. If you have a `Message`-object you wouldnt use the Logger class. Just pass is straight to the Publisher. What was your intention here?